components/organisms/Register/Register.tsx

```diff
@@ -127,6 +131,15 @@ export const Register = () => {
           {condicionalView ? (
             <>
               <div className={styles["register-form-passwordInputs"]}>
+                <div>
+                  <button
+                    className={styles["register-backButtonFormRegister"]}
+                    onClick={handleBackButtonClick}
```

Comment:
remove the handleBackButtonClick function and change state from here directly

Reply:
remove function